sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -353,7 +370,62 @@ struct graphs_support;
 } // namespace device
 } // namespace info
 
-class node {};
+enum class dynamic_parameter_type {
+  usm,
+  buffer,
```

Comment:
shall we use buffer_or_image?

Reply:
changed this to `accessor` to cover both